tox.ini

```
@@ -0,0 +1,58 @@
+[tox]
+skipsdist = True
+usedevelop = True
+skip_missing_interpreters = True
+
+envlist = 
+    python{3.7,3.8}-django{3.2}-wagtail{2.15,2.16}
+    python{3.8,3.9,3.10}-django{3.2,4.0}-wagtail{3.0,4.0}
+    python{3.9,3.10}-django{4.1}-wagtail{4.0}
+
+[gh-actions]
+python =
+    3.7: python3.7
+    3.8: python3.8
+    3.9: python3.9
+    3.10: python3.10
+
+
+[testenv]
+install_command = pip install -e ".[testing]" -U {opts} {packages}
+commands = coverage run testmanage.py test --deprecation all
```

```suggestion
commands =
    coverage run testmanage.py test --deprecation all {posargs: -v 2}
    coverage report -m
```

the latter prints the coverage results which is nice